app/models/claims/claim.rb

```diff
@@ -29,6 +29,7 @@
 #  fk_rails_...  (school_id => schools.id)
 #
 class Claims::Claim < ApplicationRecord
+  MAXIMUM_CLAIMABLE_HOURS = 20
```

Comment:
This constant is repeated in 3 places now. We'll need to consolidate to a single place soon.

Not for this PR though.